dc-cudami-server/dc-cudami-server-backend-jdbi/src/main/java/de/digitalcollections/cudami/server/backend/impl/jdbi/identifiable/entity/CollectionRepositoryImpl.java

```diff
@@ -431,14 +437,9 @@ public Collection getParent(UUID uuid) {
             + ".uuid = cc.parent_collection_uuid";
 
     Filtering filtering =
-        Filtering.defaultBuilder()
-            .filter("cc.child_collection_uuid")
-            .isEquals(new FilterValuePlaceholder(":uuid"))
-            .build();
-
-    Collection result =
-        retrieveOne(sqlSelectReducedFields, sqlAdditionalJoins, filtering, Map.of("uuid", uuid));
+        Filtering.defaultBuilder().filter("cc.child_collection_uuid").isEquals(uuid).build();
```

Comment:
add internal filter builder with safe not to be resolved column  names. not to be used in controllers / for user input / urls

Reply:
added "filterNative" to builder